test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,26 @@
 
 public class UtilsTest {
 
+    @Test
+    public void tryAnyNullBehaviour() throws Exception {
+
+        // only null objects
+        assertSomeNull(null, null);
+
+        // one null object out of a mix
+        assertSomeNull("asdf", 1, null);
+        // no null objects
+        assertNoNull("1", 1, 5);
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
+    private void assertSomeNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
Good to see you following the style of the tests nearby. 👍 
In this case, there is not much value in abstracting these two utility methods though. It does not reduce code duplication or help to clarify the intent of the code further.